commands/daily.py

```diff
@@ -15,6 +15,13 @@ def __init__(self, client):
 
     @commands.command()
     async def daily(self, ctx):
+        
+        def update_lastClaimed_and_streaks():

```

Comment:
could probably go with `update_cooldown_and_streaks` since you mentioned the mix of naming conventions